src/Fields/broadcasting_abstract_fields.jl

```diff
@@ -31,13 +31,6 @@ const BroadcastedArrayOrCuArray = Union{Broadcasted{<:DefaultArrayStyle},
     end
 end
 
-# TODO: make this support Field that are windowed in _only_ 1 or 2 dimensions.
-# Right now, this may only produce expected behavior (re: dimensionality) for
-# WindowedField that are windowed in three-dimensions. Of course, broadcasting with
-# scalar `bc` is no issue.
-@inline Base.Broadcast.materialize!(dest::WindowedField, bc::BroadcastedArrayOrCuArray) =
-    Base.Broadcast.materialize!(parent(dest), bc)
-
```

Comment:
What went wrong?

Reply:
I guess this affects that minor case when you wabt to broadcast a field and an Array (because the Array has non-shifted indexing, but the `WindowedField` has the "global" indices)

Reply:
it was a duplicated at 45-46?

Reply:
oh 🤦